pkg/commands/artifact/run.go

```diff
@@ -329,7 +329,7 @@ func (r *runner) initJavaDB(opts flag.Options) error {
 
 	// Update the Java DB
 	noProgress := opts.Quiet || opts.NoProgress
-	javadb.Init(opts.CacheDir, opts.JavaDBRepository, opts.SkipJavaDBUpdate, noProgress, opts.Insecure)
+	javadb.Init(opts.CacheDir, opts.JavaDBRepository, opts.SkipJavaDBUpdate, noProgress, ftypes.RegistryOptions{Insecure: opts.Insecure})
```

Comment:
In your case registry options (e.g. `Credentials`) are not used  for java-db.
We can build them from `opts flag.Options`:
```suggestion
	javadb.Init(opts.CacheDir, opts.JavaDBRepository, opts.SkipJavaDBUpdate, noProgress, opts.RegistryOpts())
```

Reply:
Thank you for your review. I added your suggested change.